boards/olimex-msp430-h1611/include/board-conf.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2014 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef BOARD_CONF_H
+#define BOARD_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define INFOMEM     (0x1000)
```

Comment:
any info on what this is?

Reply:
If that's a hardware property (e.g. can't be configured by the board) it should be defined by the CPU - and be documented.  

I reckon we also don't place firmware there, but instead have a special linker section for this memory?

Reply:
That would be how IMO we should do it. As of now, it is up to the application developer to use it.

This is certainly on the list of todos for cleaning up the MSP430 platform; as well as cleaning up all of the `periph` drivers. (Likely the `periph` drivers are more pressing.)

My original plan was to get in support for the Olimex board without doing any cleanup, so that I can work and test the cleanup with real hardware without having to juggle to much out of tree code. However, adding another copy of the `void msp430_init_dco(void)` function implementing the exact same thing for every board is something that just felt too wrong.

Still, I would like to keep the `INFOMEM` as a define and on board level for now. Moving that to CPU level would however be swift and easy follow-up PR that I could do prior to cleaning up the peripherals.